modules/upload_contacts_from_file.py

```diff
@@ -0,0 +1,121 @@
+import csv
+import re
+from modules import utils, visit_date_helper
+from management.models import Contact
+
+def csv_upload(filepath):
+	with open(filepath) as csvfile:
+		reader = csv.DictReader(csvfile)
+		
+		for row in reader:
+			new_dict = make_contact_dict(row)
+			
+			try:
+				new_contact = Contact.objects.get(name=new_dict["name"],phone_number=new_dict["phone_number"])
+			except Contact.DoesNotExist:
+				new_contact = Contact.objects.get_or_create(**new_dict)
+
+def make_contact_dict(row):
+	new_dict = {}
+	new_dict["name"] = row["Name"]
+	new_dict["phone_number"] = row["Phone Number"]
```

Comment:
I personally prefer to construct a dictionary like:

```
return {
    "name": row["Name"],
    "phone_number": row["Phone Number"],
    ...
```

A matter of personal preference, though.